SPECIFICATION.md

```diff
@@ -158,6 +159,73 @@ be on their own line.
 To prevent reference loops and to facilitate single-pass parsers, only backward
 references are allowed. All references glyphs must have been previously defined.
 
+#### Whitespaces
+
+Some glyphs need additional whitespace at the right of the glyph, or even
+consist of only whitespace (no polylines). The width of this whitespace can be
+specified for every glyph with the "S" character (for "space"), followed by a
+width in the same unit as polylines.
+
+The most important space glyph (which every font file MUST contain) is " "
```

Comment:
Where do we specify required glyphs?

Reply:
Not sure. Maybe it's better to remove this requirement, probably every font designer will notice anyway that a font without the space glyph is useless :grin: